rotkehlchen/globaldb/schema.py

```diff
@@ -238,6 +238,16 @@
 );
 """
 
+DB_CREATE_GENERAL_CACHE = """
+CREATE TABLE IF NOT EXISTS general_cache (
```

Comment:
Yes they are **but** that does not mean you should not add them in the DB ugprade.

All DB upgrades should contain table additions and removals. This is a bug that hit us already in the user DB upgrades and we had to fix it.

Imagine we are at version of global DB v6. And v4->v5 also removes this table, or does some kind of other modification. And a user upgrades from v2 to v6. It will go:

v2 -> v3  (but not create the new table)
v3 -> v4
v4 -> v5 (try to modify the table but the table won't exist)
----> BOOM

Reply:
Makes sense, alright